recipes/glass/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "glass" %}
+{% set version = "2024.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/glass-{{ version }}.tar.gz
+  sha256: 0041b256f4517093bf3b1671e9582134c222a18863d7ef77c656778a03927df2
+
+build:
+  # healpy does not support windows
```

Comment:
You can make this noarch. The CI on Windows will fail with a not-sovable error but we can still merge